webview-ui/src/components/chat/chat-view/components/messages/StreamingIndicator.tsx

```diff
@@ -1,22 +1,21 @@
-import React, { useMemo } from "react"
-import { ClineMessage } from "@shared/ExtensionMessage"
 import { findLast } from "@shared/array"
+import type { ClineMessage } from "@shared/ExtensionMessage"
+import { useMemo } from "react"
 
 /**
  * Hook to determine if the chat is currently streaming
  * Encapsulates the complex streaming detection logic
  */
-export const useIsStreaming = (
-	modifiedMessages: ClineMessage[],
-	clineAsk?: string,
-	enableButtons?: boolean,
-	primaryButtonText?: string,
-): boolean => {
+export const useIsStreaming = (modifiedMessages: ClineMessage[], clineAsk?: string, task?: ClineMessage): boolean => {
```

Comment:
This is NOT being called anywhere in the codebase.

Reply:
deleted